llvm/docs/NVPTXUsage.rst

```diff
@@ -911,6 +911,29 @@ including that ``wgmma.mma_async`` instruction is undefined behavior.
 For more information, refer PTX ISA
 `<https://docs.nvidia.com/cuda/parallel-thread-execution/#asynchronous-warpgroup-level-matrix-instructions-wgmma-wait-group>`_.
 
+'``llvm.nvvm.griddepcontrol.*``'
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Syntax:
+"""""""
+
+.. code-block:: llvm
+
+  declare void @llvm.nvvm.griddepcontrol.launch_dependents()
+  declare void @llvm.nvvm.griddepcontrol.wait()
+
+Overview:
+"""""""""
+
+The ``griddepcontrol`` intrinsics allows the dependent grids and prerequisite grids as defined by the runtime, to control execution in the following way:
+
+``griddepcontrol.launch_dependents`` intrinsic signals that specific dependents the runtime system designated to react to this intrinsic can be scheduled as soon as all other CTAs in the grid issue the same intrinsic or have completed. The dependent may launch before the completion of the current grid. There is no guarantee that the dependent will launch before the completion of the current grid. Repeated invocations of this intrinsic by threads in the current CTA will have no additional side effects past that of the first invocation.
+
```

Comment:
This could use some editing.


> The griddepcontrol.launch_dependents intrinsic signals that specific dependents, designated by the runtime system, can be scheduled as soon as all other CTAs in the grid issue the same intrinsic or finish executing. These dependents may launch before the current grid completes, but there is no guarantee of this.  Repeated calls to this intrinsic by threads within the same CTA have no effect after the first invocation.

Reply:
Sure, I have summarized the documentation for `launch_dependents`